Exercise04/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise04
+{
+    class Program
+    {
+        public static IEnumerable<string> GetAllFiles(string path, Func<string, bool> endWith)

```

Comment:
`endWith` should be renamed to `filter` or `condition`

Reply:
Done